sensors/jira_sensor.py

```diff
@@ -11,13 +11,14 @@ class JIRASensor(PollingSensor):
     Sensor will monitor for any new projects created in JIRA and
     emit trigger instance when one is created.
     '''
+
     def __init__(self, sensor_service, config=None, poll_interval=5):
-        super(JIRASensor, self).__init__(sensor_service=sensor_service,
-                                         config=config,
-                                         poll_interval=poll_interval)
+        super(JIRASensor, self).__init__(
```

Comment:
Would prefer to keep cosmetic changes like this to a minimum, makes the PR artificially large. I notice there are several more as well.